package.json

```diff
@@ -45,18 +45,14 @@
   "dependencies": {
     "chalk": "^4.0.0",
     "compare-versions": "^6.1.1",
-    "fs-extra": "^10.1.0",
```

Comment:
Removed packages here are needed in `test/package.json` but not needed in parent `package.json`.